src/olympia/blocklist/forms.py

```diff
@@ -39,10 +39,18 @@ class MultiAddForm(MultiGUIDInputForm):
 
     def clean(self):
         guids = splitlines(self.cleaned_data.get('guids'))
+        errors = []
         if len(guids) == 1:
             guid = guids[0]
             blk = self.existing_block = Block.objects.filter(guid=guid).first()
             if not blk and not Addon.unfiltered.filter(guid=guid).exists():
-                raise ValidationError(
+                errors.append(ValidationError(
                     _('Addon with GUID %(guid)s does not exist'),
-                    params={'guid': guid})
+                    params={'guid': guid}))
+        for guid in guids:
+            if BlockSubmission.get_submission_from_guid(guid):
+                errors.append(ValidationError(
+                    _('GUID %(guid)s is already in pending BlockSubmission'),
```

Comment:
Should the message read: "GUID %(guid)s is already in *a* pending BlockSubmission"?

Reply:
https://github.com/mozilla/addons-server/pull/13479/commits/7595cf2bf80cbc3035ebc93e7ecbe3b1947e81c3